OpenOversight/app/models.py

```diff
@@ -62,6 +63,7 @@ class Unit(db.Model):
 
     id = db.Column(db.Integer, primary_key=True)
     descrip = db.Column(db.String(120), index=True, unique=False)
+    department_id = db.Column(db.Integer)
```

Comment:
same FK comment

Reply:
Fixed in 495df5d0e783ed831e39b3893b9c9f3c954800ae